kokkos.presets.json

```diff
@@ -0,0 +1,68 @@
+{
+    "version"   : 3,
+    "cmakeMinimumRequired" : {
+        "major" : 3,
+        "minor" : 22,
+        "patch" : 0
+    },
+    "configurePresets" : [
+        {
+            "name"          : "default",
+            "displayName"   : "default",
```

Comment:
What's the point of using `displayName` if we are just echoing the name of the preset?